x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/Types.java

```diff
@@ -128,19 +128,19 @@ public class Types {
     static final ClassName RELEASABLES = ClassName.get("org.elasticsearch.core", "Releasables");
 
     static ClassName blockType(TypeName elementType) {
-        if (elementType.equals(TypeName.BOOLEAN)) {
+        if (elementType.equals(TypeName.BOOLEAN) || elementType.equals(BOOLEAN_BLOCK)) {
```

Comment:
I'm not sure `elementType` should be `BOOLEAN_BLOCK` here - it looks like in this case the method isn't doing anything - you already have the `Block` type.